lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```
@@ -1676,6 +1676,12 @@ definitions:
             maximum: 9
             x-nullable: false
 
+  converged_core_config:
```

Either change this datatype as below, or change both the type and its references to "config" naming
```suggestion
  enable_converged_core:
```